ui/e2e/src/pages/DashboardPage.ts

```diff
@@ -80,6 +81,7 @@ export class DashboardPage {
 
     this.panelEditor = page.getByTestId('panel-editor');
     this.variableEditor = page.getByTestId('variable-editor');
+    this.discardChangesConfirmationDialog = page.getByTestId('discard-changes-confirmation-dialog');
```

Comment:
You can usually get away with just looking for a dialog with the `getDialog` helper in this file and passing in the name to lean more on looking things up accessibly. Is it possible to do that for this test? If you ran into issues, the `data-testid` is an ok fallback.

Reply:
oh this makes a lot of sense since there should only be one dialog appearing at a time :)